[CAS-998] Feature - Theme for media activity (#2046)
GetStream/stream-chat-android: docusaurus/docs/Android/03-ui/02-theming.md

:::note
Your custom themes can use the default styles can use the default styles for these Views as their parent. See the [sample app's themes](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components-sample/src/main/res/values/themes.xml) for examples of this.
:::

#### Themes for Activities

> In the SDK there are also some activities: `AttachmentMediaActivity`, `AttachmentActivity` and `AttachmentGalleryActivity`. To customize those activities, it is necessary to override the activity in your manisfest: 

Reviewer: ```suggestion
SDK contains following activities: `AttachmentMediaActivity`, `AttachmentActivity` and `AttachmentGalleryActivity`. You can customize them by overriding the activity with custom `theme` in your manisfest: 
```
